MIGRATING.md

```diff
@@ -89,6 +89,22 @@ major releases of `cosmwasm`. Note that you can also view the
   +canonical_addr.as_slice()
   ```
 
+- If you use any `u128` or `i128` in storage or message types, replace them with
+  `Uint128` and `Int128` respectively to preserve the current serialization.
+  Failing to do this will result in deserialization errors!
+
+  ```diff
+  #[cw_serde]
+  struct MyStorage {
+  -  a: u128,
+  -  b: i128,
+  +  a: Uint128,
+  +  b: Int128,
+  }
+  -const map: Map<u128, MyStorage> = Map::new("map");
+  +const map: Map<Uint128, MyStorage> = Map::new("map");
```

Comment:
This example is misleading because storage keys are not JSON encoded. See `impl<'a> PrimaryKey<'a>` in storage plus. As far as I can see this is not implemented for Uint128.

Reply:
Oh, great catch. I will add a separate item instead to show that direct usage also requires replacement.